### DISCUSSION TITLE
How to use opengrok-reindex-project options

### DISCUSSION THREAD

**Discussion by dkr91** [oracle/opengrok#4554] [Q&A] [answered]
When using opengrok-reindex-project it asked for options, but there is no documentation I could find that list those. Without these I can't use this script

**Comment by vladak** [comment]
How did you run it and what was the exact error message ?

**Reply by dkr91** [reply]
> opengrok-reindex-project -p path/to/project
> usage: /usr/local/bin/opengrok-reindex-project [-h] [-l LOGLEVEL] [-v] [-j JAVA] [-J JAVA_OPTS] [-e ENVIRONMENT] [--doprint boolean] (-a JAR | -c CLASSPATH) [-t TEMPLATE] [-p PATTERN] -P PROJECT [-d DIRECTORY] [-U URI] [--printoutput] [-H HEADER [HEADER ...]] [--api_timeout API_TIMEOUT] options [options ...]
> /usr/local/bin/opengrok-reindex-project: error: the following arguments are required: options

I can't find the option opengrok-reindex-project wants to have

**Reply by vladak** [reply, marked as answer]
These are indexer options. See https://github.com/oracle/opengrok/wiki/Per-project-management-and-workflow#opengrok-sync for inspiration.
